js/m.setup.js

```diff
@@ -269,16 +268,17 @@ function setupLandmarks(landmarks) {
 
 function postSetup(model) {
   var formattedModel = {
-    'model': model['model_url'],
-    'meshes': model['mesh_url'],
-    'landmarks': model['landmark_url'],
+    'model': model['model'],
+    'meshes': model['mesh'],
+    'landmarks': model['landmark'],
     'showmeshes': model['showmeshes'],
-    'targetURL': model['target-url']
+    'targetURL': model['target'],
```

Comment:
Even internally to the code `targetURL` should be renamed `target`. The anchor tag's `target` attribute is a name of a context rather than a URL. https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a